scripts/build_local.sh

```diff
@@ -31,6 +31,23 @@ if [ -n "${USE_ANACONDA}" ]; then
   export CONDA_INSTALL_LOCALLY=1
   "${ROOT_DIR}/scripts/build_anaconda.sh" "$@"
 else
+  # Make sure that pyyaml is installed for the codegen of building Aten to work
+  if [[ -n "$(python -c 'import yaml')" ]]; then
+    if [[ -x "$(which pip)" ]]; then
+      # Use pip if it's available
+      echo "Installing pyyaml with pip at $(which pip)"
+      pip install pyyaml
+    else
+      # Otherwise use the packages setup.py
+      pyyaml_dir=_temp_pyyaml_install
+      mkdir $pyyaml_dir && pushd $pyyaml_dir
+      curl http://pyyaml.org/download/pyyaml/PyYAML-3.12.tar.gz --output tarfile
+      tar -xf tarfile --strip 1
+      sudo python setup.py install
```

Comment:
`sudo` ??

Also, what is the case when `pip` isn't available?

Reply:
We have CI builds for conda python, homebrew python, and "system" python /usr/bin that doesn't come with pip

Reply:
Actually I'm going to change this to first install pip with easy_install and then use that. That's probably closer to what users will have, and if they want a useful python installation they'll probably need pip anyways.